docs/New-Maintainer-Checklist.md

```diff
@@ -53,36 +53,57 @@ Thanks for all your work so far!
 
-If they are elected to the Homebrew's [Project Leadership Committee](https://docs.brew.sh/Homebrew-Governance#4-project-leadership-committee):
+If there are problems, ask them to step down as a maintainer.
 
-- Make them [owners on the Homebrew GitHub organisation](https://github.com/orgs/Homebrew/people)
-- Invite them to the [**@Homebrew/plc** team](https://github.com/orgs/Homebrew/teams/plc/members)
-- Invite them to [Google Analytics](https://analytics.google.com/analytics/web/#management/Settings/a76679469w115400090p120682403/%3Fm.page%3DAccountUsers/).
-- Invite them to the [`homebrew` private 1Password](https://homebrew.1password.com/people).
-- Make them owners on the [`machomebrew` private maintainers Slack](https://machomebrew.slack.com/admin).
-
-If there are problems, ask them to step down as a maintainer and revoke their access to all of the above.
+When they step down as a maintainer for any reason, revoke their access to all of the above.
 
 In the interests of loosely verifying maintainer identity and building camaraderie, if you find yourself in the same town (e.g living, visiting or at a conference) as another Homebrew maintainer you should make the effort to meet up. If you do so, you can [expense your meal](https://docs.opencollective.com/help/expenses-and-getting-paid/submitting-expenses) (within [Homebrew's reimbursable expense policies](https://opencollective.com/homebrew/expenses)). This is a more relaxed version of similar policies used by other projects, e.g. the Debian system to meet in person to sign keys with legal ID verification.
 
 Now sit back, relax and let the new maintainers handle more of our contributions.
 
+## PLC
+
+If a maintainer or member is elected to the Homebrew's [Project Leadership Committee](https://docs.brew.sh/Homebrew-Governance#4-project-leadership-committee):
+
+- Invite them to the [**@Homebrew/plc** team](https://github.com/orgs/Homebrew/teams/plc/members)
+- Make them [billing managers](https://github.com/organizations/Homebrew/settings/billing) and [moderators](https://github.com/organizations/Homebrew/settings/moderators) on the Homebrew GitHub organisation
+
+When they step down as a PLC member, revoke or downgrade their access to all of the above.
+
+## TSC
+
+If a maintainer is elected to the Homebrew's [Technical Steering Committee](https://docs.brew.sh/Homebrew-Governance#7-technical-steering-committee):
+
+- Invite them to the [**@Homebrew/tsc** team](https://github.com/orgs/Homebrew/teams/tsc/members)
+- Make them [billing managers](https://github.com/organizations/Homebrew/settings/billing) and [moderators](https://github.com/organizations/Homebrew/settings/moderators) on the Homebrew GitHub organisation
+
+When they step down as a TSC member, revoke or downgrade their access to all of the above.
+
+## Owners
+
+The Project Leader, one other PLC member (ideally a maintainer) and one other TSC member should be made owners on GitHub and Slack:
```

Comment:
This uses "...should be made owners on GitHub and Slack" language but the following list also includes 1Password. Should we use "...GitHub, Slack, and 1Password" language here or simply omit the specifics and let the list do the work (e.g., something like "...should be made owners of the following:" or "...should be made owners where appropriate:" along with adapting the list items to remove the repeated "Make them owners on the..." text). Either would work but the latter may be cleaner and easier to maintain.